tests/sync/conftest.py

```diff
@@ -121,3 +127,71 @@ async def task() -> None:
         return list(map(lambda action: results[action], actions))
 
     yield _sync_gather_impl
+
+
+class TraceViewerPage:
+    def __init__(self, page: Page):
+        self.page = page
+
+    @property
+    def actions_tree(self) -> Locator:
+        return self.page.get_by_test_id("actions-tree")
+
+    @property
+    def action_titles(self) -> Locator:
+        return self.page.locator(".action-title")
+
+    @property
+    def stack_frames(self) -> Locator:
+        return self.page.get_by_test_id("stack-trace-list").locator(".list-view-entry")
```

Comment:
It would be great to update trace viewer, so this becomes `get_by_role("list", name="Stack Trace").get_by_role("listitem")`.